src/linear_algebra/bilinear_form.lean

```diff
@@ -1223,6 +1318,11 @@ lemma to_dual_def {B : bilin_form K V} (hB : B.nondegenerate) {m n : V} :
 
 end
 
+/- We note that we cannot use `restrict_nondegenerate_iff_is_compl_orthogonal` for the lemma below
```

Comment:
If you write a comment like this, it should show up in the generated documentation pages.
```suggestion
/-! We note that we cannot use `restrict_nondegenerate_iff_is_compl_orthogonal` for the lemma below
```

Reply:
Is the lemma `_root_.restrict_nondegenerate_iff_is_compl_orthogonal` or is it something like `bilin_form.restrict_nondegenerate_iff_is_compl_orthogonal`? If it's not `_root_`, make sure to put the prefix in the `` ` ``, so that it generates a link.